src/SFA.DAS.EmployerIncentives.Api.UnitTests/Controllers/PassThroughControllerTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerIncentives.Api.Controllers;
+using SFA.DAS.EmployerIncentives.Infrastructure.Api;
+using SFA.DAS.EmployerIncentives.Interfaces;
+using SFA.DAS.EmployerIncentives.Models.PassThrough;
+
+namespace SFA.DAS.EmployerIncentives.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public class PassThroughControllerTests
+    {
+        [Test]
+        public async Task When_AddLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_is_called()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForAddLegalEntityCall();
+
+            await f.Sut.AddLegalEntity(f.AccountId, f.LegalEntityRequest);
+
+            f.EmployerIncentivesPassThroughServiceMock.Verify(x => x.AddLegalEntity(f.AccountId, f.LegalEntityRequest));
+        }
```

Comment:
Bit of an interesting one this...we probably need to make sure the same conventions and tool are used across the solution even though that may differ from what we have agreed on EI.

Reply:
Ok will discuss with the Dan